Contributors.md

```diff
@@ -233,4 +233,17 @@
 <!-- End of column-18 -->
 </tr>
 </table>
-
+<!-- Start of column-19 -->
```

Comment:
Kindly add your profile into the table under the `<!-- End of coulmn-18 -->` line and delete the duplicate `</table>` and `</tr>` tags

So change will look like this

```yml
<!-- End of column-18 -->
<!-- Start of column-19 -->
<td align="center">
  <a href="https://github.com/winniffy">
    <img
      src="https://avatars.githubusercontent.com/u/81039726?v=4"
      width="100px"
    />
    <br />
    <sub>Winner Umeh</sub>
  </a>
</td>
<!-- End of column-19 -->
```